tests/unit/sagemaker/jumpstart/constants.py

```diff
@@ -1620,6 +1620,172 @@
             },
         },
     },
+    "default_payloads": {
+        "model_id": "model-depth2img-stable-diffusion-v1-5-controlnet-v1-1-fp16",
+        "url": "https://huggingface.co/lllyasviel/control_v11f1p_sd15_depth",
+        "version": "1.0.0",
+        "min_sdk_version": "2.144.0",
+        "training_supported": False,
+        "incremental_training_supported": False,
+        "hosting_ecr_specs": {
+            "framework": "djl-deepspeed",
+            "framework_version": "0.21.0",
+            "py_version": "py38",
+            "huggingface_transformers_version": "4.17",
+        },
+        "hosting_artifact_key": "stabilityai-infer/infer-model-depth2img-st"
+        "able-diffusion-v1-5-controlnet-v1-1-fp16.tar.gz",
```

Comment:
sanity-check: did you intend to jump a line here?

Reply:
this was just to reduce the line length